defaults/main.yml

```diff
@@ -0,0 +1,45 @@
+www_data_home_dir: '/var/www'
+accounting_screenrc_path: '/home/ubuntu/accounting-screenrc'
+
+ACCOUNTING_DOMAIN_NAME: '{{ inventory_hostname }}'
+
+# Contents of accounting app's ssl certificate
+ACCOUNTING_SSL_CERT: null
+ACCOUNTING_SSL_KEY: null
```

Comment:
Oh, yeah that would be good to remove them then. Or maybe since this is public I should mention to the user that these need to be setup manually with `certbot` since the playbook won't do it.